versions.tf

```
@@ -1,15 +1,10 @@
 terraform {
-  required_version = ">= 0.13"
+  required_version = ">= 1.0"
```

```suggestion
  required_version = ">= 0.15.0"
```

do we need to restrict the module to TF 1.x.x?
Maybe the callers of the module could restrict it?

---

Updated